etl/steps/export/multidim/wb/latest/world_bank_pip.config.yml

```diff
@@ -0,0 +1,103 @@
+title:
+  title: Poverty indicators
+  title_variant: World Bank
+default_selection:
+  - Bangladesh
+  - Bolivia
+  - Madagascar
+  - India
+  - China
+  - Ethiopia
+
+dimensions:
+  - name: Metric
+    slug: indicator
+    choices:
+      - slug: headcount_ratio
+        name: Share in poverty
+        description: The share of the population living below the selected poverty line.
+      - slug: headcount
+        name: Number in poverty
+        description: The number of people living below the selected poverty line.
+      # - slug: total_shortfall
+      #   name: Total shortfall from poverty line
+      # - slug: avg_shortfall
+      #   name: Average shortfall ($ per day)
+      # - slug: income_gap_ratio
+      #   name: Average shortfall (% of poverty line)
+      # - slug: poverty_gap_index
+      #   name: Poverty gap index
+  - name: Poverty line
+    slug: poverty_line
+    description: In international dollars
+    choices:
+      - slug: "100"
+        name: $1 a day
+        group: Individual poverty lines
+      - slug: "215"
+        name: $2.15 a day
+        description: The international poverty line used to measure extreme poverty
+        group: Individual poverty lines
+      - slug: "365"
+        name: $3.65 a day
+        description: Set by the World Bank to reflect national definitions in lower-middle income
+        group: Individual poverty lines
+      - slug: "685"
+        name: $6.85 a day
+        description: Set by the World Bank to reflect national definitions in upper-middle income countries
+        group: Individual poverty lines
+      - slug: "1000"
+        name: $10 a day
+        group: Individual poverty lines
+      - slug: "2000"
+        name: $20 a day
+        group: Individual poverty lines
+      - slug: "3000"
+        name: $30 a day
+        group: Individual poverty lines
+      - slug: "4000"
+        name: $40 a day
+        group: Individual poverty lines
+      - slug: line_chart
+        name: Line chart
+        description: See how poverty has changed over time using 4 different poverty lines
+        group: Visualize multiple poverty lines
+      # - slug: multiple_lines_absolute
+      #   name: Multiple lines
+      # - slug: 40% of the median
+      #   name: 40% of the median
+      #   description: Relative poverty
+      # - slug: 50% of the median
+      #   name: 50% of the median
+      #   description: Relative poverty
+      # - slug: 60% of the median
+      #   name: 60% of the median
+      #   description: Relative poverty
+      # - slug: multiple_lines_relative
+      #   name: Multiple lines
+      #   description: Relative poverty
+  - name: Household survey data type
+    slug: table
+    choices:
+      - slug: Income or consumption consolidated
+        name: Show data for both income and consumption
+      - slug: Income
```

Comment:
@pabloarosado I wanted to try to have a multi-indicator view from yaml to understand how it works. This is executed, but doesn't change the output. Do you know why?